simespresso/io_QE/tests/test_start_espresso.py

```diff
@@ -0,0 +1,24 @@
+
+import unittest
+import os
+from simespresso.io_QE import espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
```

Comment:
ok - will put all tests in test_qe.py 
as for start_qe - shouldnt that be in the espresso_class, since other simphony modules will be wanting to call it ?